simple_audit/models.py

```diff
@@ -93,7 +93,7 @@ class AuditRequest(models.Model):
 
     THREAD_LOCAL = threading.local()
 
-    request_id = models.CharField(max_length=255)
+    request_id = models.CharField(max_length=255, db_index=True)
```

Comment:
@kylowrobelek if we don't filter by request_id anymore then we do not need index, right?